metabase/metabase
# Remove unsafe-inline from style-src

Related to https://github.com/metabase/metabase-private/issues/73

An integration branch with all the fixes to remove `unsafe-inline` from our CSP headers for `style-src`. Instead of `unsafe-inline` we use `nonce-`, which should be different on every request.

How to verify:
- Run the app, make sure it looks normal
- There should be no `unsafe-inline` CSP header coming from the backend
